.gitignore

```diff
@@ -17,6 +17,7 @@ _version.py
 
 # wasm process related
 node_modules
+_wasm_build
```

Comment:
what create this specically ?

Reply:
Myself, see `package.json` and `build.sh`.  
I can rename the folder, but it has to be in the source dir.

Reply:
noted